src/react/components/pages/homepage/homePage.tsx

```diff
@@ -139,8 +170,16 @@ export default class HomePage extends React.Component<IHomepageProps> {
     }
 
     private loadSelectedProject = async (project: IProject) => {
-        await this.props.actions.loadProject(project);
-        this.props.history.push(`/projects/${project.id}/edit`);
+        console.log(project);
```

Comment:
I don't think that this method should require changing... any project passed into this method should already be a valid v2 project and should open the same as any other project.